crates/factor-key-value-sqlite/src/lib.rs

```diff
@@ -0,0 +1,72 @@
+use std::{
+    fs,
+    path::{Path, PathBuf},
+};
+
+use anyhow::Context;
+use serde::{Deserialize, Serialize};
+use spin_factor_key_value::MakeKeyValueStore;
+use spin_key_value_sqlite::{DatabaseLocation, KeyValueSqlite};
+
+pub struct SpinKeyValueStore {
+    base_path: PathBuf,
+}
+
+impl SpinKeyValueStore {
```

Comment:
Its confusing that the crate is `-sqlite` but the store is `Spin`. Given the runtime config `type = "spin"` maybe we can live with `spin-factor-key-value-spin`?

Reply:
Good call. Just adjusted it.